.buildkite/pipeline.yml

```diff
@@ -13,10 +13,11 @@ steps:
   #################
   - label: "🧪 Build and Test"
     key: "test"
-    command: |
-      build_and_test_pod
+    command: .buildkite/build-and-test.sh
     artifact_paths:
-      - "fastlane/test_output/*.xcresult.zip"
+      - fastlane/test_output/*.xcresult.zip
```

Comment:
(Not so) Fun fact: This file had never been uploaded because there had never been an instruction to create it from the `.xcresult`